apps/docs/package.json

```diff
@@ -24,6 +24,7 @@
     "next-contentlayer": "^0.3.2",
     "prettier": "^2.8.8",
     "react": "18.2.0",
+    "react-collapsible": "^2.10.0",
```

Comment:
This package is not used, it must be removed.

Reply:
It's used in the sidebar 
![image](https://github.com/workleap/wl-untitled/assets/361632/02db9d1d-8008-413d-b5aa-676ebce99176)

Reply:
it comes from the package `@radix-ui/collapsible` and not from `react-collapsible`.

Reply:
My mistake